### ISSUE TITLE
Consider adding aria-keyshortcuts to the list of Translatable States and Properties

### ISSUE THREAD

**Issue by laurie-j** [w3c/aria#1910]
# Describe your concern
Since keyboards may be localized, it seems that the aria-keyshortcuts attribute should be added to the list of [Translatable States and Properties](https://www.w3.org/TR/wai-aria-1.2/#translatable-states-and-properties). Adding it to the list may make it more apparent that the string value for the aria-keyshortcuts attribute should also be localized.
# Link to the version of the specification or documentation you were looking at at.
Link to documentation:
[ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/)

Does the issue exist in [the editor's draft](https://w3c.github.io/aria/) (the editor's draft is the most recent draft of the specification)?
Yes, this issue also exists in the editor's draft

**Comment by jnurthen** [comment]
This would need to be a separate section in that an author would likely want to use different shortcuts in different languages but we would never want automated translation to attempt to modify these.
